deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -26,19 +26,19 @@
             "value": "2680"
         }, {
             "name": "PFB_MAINTENANCE_WORK_MEM",
-            "value": "4092MB"
+            "value": "8GB"
         }, {
             "name": "PFB_MAX_WAL_SIZE",
-            "value": "4096MB"
+            "value": "8GB"
         }, {
             "name": "PFB_SHARED_BUFFERS",
-            "value": "7168MB"
+            "value": "24GB"
         }, {
             "name": "PFB_TEMP_FILE_LIMIT",
-            "value": "26214400"
+            "value": "100GB"
         }, {
             "name": "PFB_WORK_MEM",
-            "value": "3072MB"
+            "value": "3GB"
```

Comment:
Yeah, I was going to raise that one,too, but it's the one that gets allocated for every thread in parallel processes so it can actually balloon pretty fast. So I left it, but still changed the format to match the others.